content/2016.09.jedit-trackers-with-kinto-en.md

```diff
@@ -0,0 +1,403 @@
+Title: Getting jEdit Trackers off the Ground using Kinto and Kinto.js
```

Comment:
Ah I see. I misunderstood the French version. I like what you were going for. I'm not sure how to attach "getting off the ground" to Kinto while also keeping emphasis on the bugs, since they're the point of the article. How about "jEdit bugs: Getting off the Ground with Kinto and Kinto.js"?